weave-js/src/components/PagePanelComponents/Home/Home.tsx

```diff
@@ -242,7 +243,7 @@ const HomeComp: FC<HomeProps> = props => {
     if (isLocallyServed && REDIRECT_LOCAL.includes(pathname)) {
       return <Redirect to={urlLocalBoards()} />;
     }
-    // This should never happen
+    // This can happen if there is an authentication problem.
```

Comment:
This in theory should only happen when weave is in server mode and running in dev mode and auth error. In prod, this should never happen as we should be redirected to the login screen